lib/ronad/just_one.rb

```diff
@@ -1,7 +1,12 @@
 module Ronad
+  # @deprecated
+  # @see Ronad::Maybe::from_multiple_values
   class JustOne < Just
-    def initialize(*args)
-      super args.detect { |v| Monad === v ? v.value : v }
+    def self.new(*args)
+      warn "[DEPRECATION] Ronad::JustOne is deprecated, see Ronad::Many::from_multiple_values."
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.

[Read more about it here.](http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/StringLiterals)